nemo/utils/exp_manager.py

```diff
@@ -189,6 +189,13 @@ def _on_batch_start(self, name):
         if self.timer.buffer_size <= 0:
             self.timer.reset(name)
 
+        if self.timer.is_active(name):
+            logging.warning(
+                f"Timer `{name}` was not correctly stopped, suggesting a "
+                "possible issue. The timer will be reset for now."
+            )
+            self.timer.reset(name)
```

Comment:
There's a reset just above this code ?

Reply:
There is a bug when during validation that the active timer didn't properly turn off. Added this WAR to fix the issue

Reply:
Ok